### ISSUE TITLE
fix repo debian snapshot container build

### ISSUE THREAD

**Issue by nkraetzschmar** [gardenlinux/repo-debian-snapshot#1]
currently snapshot build breaks at container build stage

**Comment by nkraetzschmar** [comment]
initial investigation suggests issue with debootstrap and debian testing in general:

```
# debootstrap testing chroot
I: Target architecture can be executed
I: Retrieving InRelease
I: Checking Release signature
I: Valid Release signature (key id 4CB50190207B4758A3F73A796ED0E7B82643E131)
I: Retrieving Packages
I: Validating Packages
I: Resolving dependencies of required packages...
I: Resolving dependencies of base packages...
I: Checking component main on http://deb.debian.org/debian...
I: Retrieving adduser 3.137
I: Validating adduser 3.137
I: Retrieving apt 2.9.2
I: Validating apt 2.9.2
[...]
I: Unpacking vim-tiny...
I: Unpacking whiptail...
W: Failure while installing base packages.  This will be re-attempted up to five times.
W: See /tmp/chroot/debootstrap/debootstrap.log for details (possibly the package /var/cache/apt/archives/cron_3.0pl1-189_arm64.deb is at fault)
W: Failure while installing base packages.  This will be re-attempted up to five times.
W: See /tmp/chroot/debootstrap/debootstrap.log for details (possibly the package /var/cache/apt/archives/cron_3.0pl1-189_arm64.deb is at fault)
W: Failure while installing base packages.  This will be re-attempted up to five times.
W: See /tmp/chroot/debootstrap/debootstrap.log for details (possibly the package /var/cache/apt/archives/cron_3.0pl1-189_arm64.deb is at fault)
W: Failure while installing base packages.  This will be re-attempted up to five times.
W: See /tmp/chroot/debootstrap/debootstrap.log for details (possibly the package /var/cache/apt/archives/cron_3.0pl1-189_arm64.deb is at fault)
W: Failure while installing base packages.  This will be re-attempted up to five times.
W: See /tmp/chroot/debootstrap/debootstrap.log for details (possibly the package /var/cache/apt/archives/cron_3.0pl1-189_arm64.deb is at fault)
```

So it may be an issue with debian / debootstrap upstream

**Comment by nkraetzschmar** [comment]
fixed by [34b01b2](https://github.com/gardenlinux/repo-debian-snapshot/commit/34b01b20a1ff23c3f4430f620ca531b32dad5785) (updated builder uses mmdebstrap instead of debootstrap => resolve dependencies correctly)
